source/broadband/webpa_notification.c

```diff
@@ -282,6 +291,98 @@ void FactoryResetCloudSyncTask()
 		WalInfo("FactoryResetCloudSync Thread created Successfully\n");
 	}
 }
+void *SyncNotifyRetry()
```

Comment:
Instead of adding these 2 functions in between FactoryResetCloudSync functions, move it after FactoryResetCloudSync() 

SyncNotifyRetry()
 SyncNotifyRetryTask()